halo2_proofs/src/dev/failure.rs

```diff
@@ -402,9 +459,39 @@ fn render_lookup<F: FieldExt>(
         expr.evaluate(
             &|_| panic!("no constants in table expressions"),
             &|_| panic!("no selectors in table expressions"),
```

Comment:
I't a bit weird to lookup using a Selector column or a Constant expr. But I can add support if you think it can happen. I thought it made no sense. But maybe I'm overseeing some cases.